### ISSUE TITLE
Cannot find module './config/npm-config'

### ISSUE THREAD

**Issue by davidtaylorjr** [electron/electron#8346]
I was told to repost this issue on this repository:

https://github.com/Glavin001/atom-beautify/issues/1444#issuecomment-270683262

Can't run atom-beauify says failed to activate.

... edit style.css
...
Atom Version: 1.12.9
Electron Version: 1.3.13
System: Microsoft Windows 10 Home
Thrown From: atom-beautify package, v0.29.16

Stack Trace

Failed to activate the atom-beautify package

At Cannot find module './config/npm-config'

Error: Cannot find module './config/npm-config'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\module-cache.js:383:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (C:\Users\davidtaylorjr\.atom\packages\atom-beautify\node_modules\winston\lib\winston\config.js:48:17)
    at Module._compile (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .js] (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (C:\Users\davidtaylorjr\.atom\packages\atom-beautify\node_modules\winston\lib\winston\common.js:15:14)
    at Module._compile (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .js] (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (C:\Users\davidtaylorjr\.atom\packages\atom-beautify\node_modules\winston\lib\winston.js:24:26)
    at Module._compile (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .js] (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:50:27)
    at file:///C:/Users/davidtaylorjr/.atom/packages/atom-beautify/src/logger.coffee:10:13
    at Object.<anonymous> (file:///C:/Users/davidtaylorjr/.atom/packages/atom-beautify/src/logger.coffee:4:18)
    at Object.<anonymous> (file:///C:/Users/davidtaylorjr/.atom/packages/atom-beautify/src/logger.coffee:1:1)
    at Module._compile (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .coffee] (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (file:///C:/Users/davidtaylorjr/.atom/packages/atom-beautify/src/beautifiers/index.coffee:6:10)
    at Object.<anonymous> (file:///C:/Users/davidtaylorjr/.atom/packages/atom-beautify/src/beautifiers/index.coffee:1:1)
    at Module._compile (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .coffee] (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (file:///C:/Users/davidtaylorjr/.atom/packages/atom-beautify/src/beautify.coffee:9:15)
    at Object.<anonymous> (file:///C:/Users/davidtaylorjr/.atom/packages/atom-beautify/src/beautify.coffee:2:1)
    at Module._compile (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.value [as .coffee] (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\compile-cache.js:201:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\package.js:747:27)
    at Package.module.exports.Package.activateNow (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\package.js:174:16)
    at C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\package.js:827:25
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\node_modules\event-kit\lib\emitter.js:129:28)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\command-registry.js:240:20)
    at C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:587:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:382:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\window-event-handler.js:106:36)
    at HTMLDocument.<anonymous> (C:\Users\davidtaylorjr\AppData\Local\atom\app-1.12.9\resources\app.asar\src\window-event-handler.js:3:59)
Commands

     -8:05.1.0 atom-live-server:start-3000 (atom-text-editor.editor.is-focused)
     -0:23.8.0 core:select-all (atom-text-editor.editor.is-focused)
     -0:15 core:backspace (atom-text-editor.editor.is-focused)
     -0:13.4.0 core:save (atom-text-editor.editor.is-focused)
Config

{
  "core": {
    "autoHideMenuBar": true,
    "telemetryConsent": "limited",
    "themes": [
      "atom-dark-ui",
      "seti-syntax"
    ]
  }
}
Installed Packages

# User
atom-beautify, v0.29.16 (inactive)
atom-live-server, v1.2.5 (active)
atom-monokai, v0.10.4 (inactive)
auto-detect-indentation, v1.3.0 (active)
bootstrap3-snippets, v0.1.2 (active)
color-picker, v2.2.3 (active)
console-log, v1.9.0 (active)
css-snippets, v1.1.0 (active)
emmet, v2.4.3 (active)
file-icons, v2.0.7 (active)
git-plus, v7.0.4 (active)
git-time-machine, v1.5.4 (inactive)
highlight-selected, v0.11.2 (active)
javascript-snippets, v1.2.1 (active)
linter, v1.11.18 (active)
linter-csslint, v1.3.3 (active)
linter-erb, v1.0.3 (active)
linter-htmlhint, v1.3.1 (active)
linter-javac, v1.9.4 (active)
linter-jshint, v3.0.2 (active)
linter-jsonlint, v1.3.0 (active)
linter-perl, v0.8.0 (active)
linter-php, v1.3.1 (active)
linter-python, v3.0.5 (active)
linter-ruby, v1.2.4 (inactive)
monokai-dark, v1.3.0 (inactive)
pigments, v0.37.0 (active)
seti-syntax, v1.0.1 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (active)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.45.0 (inactive)
base16-tomorrow-dark-theme, v1.3.0 (inactive)
base16-tomorrow-light-theme, v1.3.0 (inactive)
one-dark-ui, v1.6.2 (inactive)
one-light-ui, v1.6.2 (inactive)
one-dark-syntax, v1.5.0 (inactive)
one-light-syntax, v1.5.0 (inactive)
solarized-dark-syntax, v1.0.5 (inactive)
solarized-light-syntax, v1.0.5 (inactive)
about, v1.7.0 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.13.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.4 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.42.0 (active)
bracket-matcher, v0.82.2 (active)
command-palette, v0.39.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.202.2 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.158.8 (active)
metrics, v1.1.2 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.1 (inactive)
settings-view, v0.243.1 (active)
snippets, v1.0.4 (active)
spell-check, v0.68.4 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.2 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.0 (active)
timecop, v0.33.2 (active)
tree-view, v0.210.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.35.0 (active)
wrap-guide, v0.38.2 (active)
language-c, v0.54.0 (active)
language-clojure, v0.22.1 (active)
language-coffee-script, v0.48.0 (active)
language-csharp, v0.13.0 (active)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.43.0 (active)
language-html, v0.47.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (active)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.37.0 (active)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (active)
language-ruby, v0.70.2 (active)
language-ruby-on-rails, v0.25.1 (active)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (active)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages

**Comment by kevinsawicki** [comment]
Closing this out based on the updated comment at https://github.com/Glavin001/atom-beautify/issues/1444#issuecomment-270683262

> Update: Nope. The error is originating from Winston and being throw from Electron. See:

Electron is just passing through this error, not generating it.

This is the line it originates from 👇 

```
at Object. (C:\Users\davidtaylorjr.atom\packages\atom-beautify\node_modules\winston\lib\winston\config.js:48:17)
```

**Comment by davidtaylorjr** [comment]
Ok thanks, back to the drawing board.
On 1/5/2017 12:22:50 PM, Kevin Sawicki <notifications@github.com> wrote:
Closed #8346 [https://github.com/electron/electron/issues/8346].
